src/main/scheduler/scheduler.c

```diff
@@ -535,6 +546,10 @@ FAST_CODE void scheduler(void)
                 // Total tasks run in last second
                 DEBUG_SET(DEBUG_TIMING_ACCURACY, 3, taskCount);
 
+                lateTaskPercentage = 1000 * (uint32_t)lateTaskCount / taskCount;
```

Comment:
```suggestion
                lateTaskPercentage = (uint32_t) 1000 * lateTaskCount / taskCount;
```

Reply:
Why?

Reply:
Not important - was wondering if this would be the same result.

Reply:
` lateTaskPercentage = 1000 * lateTaskCount / taskCount;` ?